modules/1_opencv/src/opencv_coins.cpp

```diff
@@ -11,4 +14,25 @@ unsigned countCoins(const Mat& img) {
 
     // TODO: implement an algorithm from https://docs.opencv.org/master/d3/db4/tutorial_py_watershed.html
     CV_Error(Error::StsNotImplemented, "countCoins");
```

Comment:
You need to remove this assertion

Reply:
I did it

Reply:
Where?

Reply:
It was the answer to 
Please check https://github.com/dkurt/openvino_practice/pull/85/files - the changes are the same